task7/src/components/movieList/movieList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
+import './movieList.scss';
+import Movie from '../movie/movie';
+
+function MovieList({movies = []}) {
+  return (
+    <div className="movie-list">
+      <span className="movies-count"><b>{movies.length}</b> Movies found</span>
+      <div className="movie-container">
+        { movies.length ?
+          movies.map(movie => (<Movie key={`mov${movie.id}`} movie={movie}/>)) :
+          <h2>No Movie Found</h2> }
+      </div>
+    </div>
+  );
+}
+
+const mapStatetoProps = (state) => {
+  const {movies} = state;
+  return {movies}
```

Comment:
I am only curious... why do you return it into curly brackets?

Reply:
The docs said that we should return a plain object with the component's prop https://react-redux.js.org/using-react-redux/connect-mapstate#return